test/functional/cmdLineTests/jvmtitests/playlist.xml

```diff
@@ -207,6 +207,36 @@
 			<type>native</type>
 		</types>
 	</test>
+
+
+	<test>
+		<testCaseName>cmdLineTester_fieldwatchtests</testCaseName>
+		<variations>
+		</variations>
+		<command>$(ADD_JVM_LIB_DIR_TO_LIBPATH) \
+	$(JAVA_COMMAND) $(JVM_OPTIONS) -Xshareclasses:none \
+	-DTEST_ROOT=$(Q)$(TEST_RESROOT)$(Q) \
+	-DJAR=$(Q)$(TEST_RESROOT)$(D)jvmtitest.jar$(Q) \
+	-DEXE=$(SQ)$(JAVA_COMMAND) $(JVM_OPTIONS) -Xdump $(SQ) \
+	-DMODE_HINTS=$(Q)$(MODE_HINTS)$(Q) \
+	-jar $(CMDLINETESTER_JAR) \
+	-config $(Q)$(TEST_RESROOT)$(D)fieldwatchtests.xml$(Q) \
+	-explainExcludes \
+	-xids all,$(PLATFORM),$(JCL_VERSION) -xlist $(Q)$(TEST_RESROOT)$(D)jvmtitests_excludes_$(JDK_VERSION).xml$(Q) -nonZeroExitWhenError; \
+	${TEST_STATUS}</command>
+		<platformRequirements>arch.390,arch.x86</platformRequirements>
```

Comment:
What platforms do you expect these tests to run on?  These platform requirements are &&, so I am not sure this will do what you expect (arch.390 && arch.x86) will not ever be run.  cmdLineTester_fieldwatchtests was skipped on xLinux: https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_extended.functional_x86-64_linux_Personal/7/tapResults/

Reply:
Ohh. I didn't realize that. Can't remember if I added these before or after I finished running these tests locally because the tests were running on my Z setup.

So how would I specify that I want to run on Z and X? 
```
<platformRequirements>arch.390</platformRequirements>
<platformRequirements>arch.x86</platformRequirements>
```
Something like the above? Or does it need to be a list of platforms NOT to run on?
```
<platformRequirements>^Power,^aarch etc...</platformRequirements>
```

Reply:
playlists do not support multiple platformRequirement tags within test tags, so your first guess won't work :)

Think if on z and x, could do
`<platformRequirements>^arch.ppc,^arch.arm</platformRequirements>`
so this will include os.osx, os.win, os.linux 

see an example of using ^ here:
https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/systemtest/otherLoadTest/playlist.xml#L343

Reply:
Thanks! I've addressed this in 03610df for you to verify. I'll do a squash + rebase of all commits to address review comments once you've approved the PR.